conferences/tests.py

```diff
@@ -9,59 +9,23 @@
 from users.models import Organization
 from .models import Zosia, Place, UserPreferences, Bus
 from .forms import UserPreferencesForm
+from .test_helpers import new_zosia, new_user, new_bus, PRICE_ACCOMODATION, PRICE_BASE, \
```

Comment:
You can move this import to something like that `form xyz import (....)` - that way you don't need `\`